src/workerd/server/workerd.c++

```diff
@@ -669,6 +672,19 @@ public:
     server.overrideExternal(kj::mv(name), kj::str(value));
   }
 
+  void enableInspector(kj::StringPtr param) {
+    // The param can be either a numeric socket fd or a socket address.
+    KJ_IF_MAYBE(fd, param.tryParseAs<uint>()) {
```

Comment:
So for sockets, `--socket-addr` and `--socket-fd` are two different flags, but here you're using a single flag and assuming it's an FD if it parses successfully as an integer. This feels a little inconsistent.

Do you think anyone will actually want to pass an inherited FD here? The purpose of that for sockets is so that systemd can take care of opening the socket on a privileged port, and pass the same socket to successive restarts of the service. But it feels unlikely to me that people would have systemd manage the inspector port for them, so maybe this is unnecessary code?

Reply:
Honestly, I don't know and I went back and forth on this. I think it should be fine to limit this to an address and remove the fd bit for now and add that back in later if someone asks for it.

Reply:
I went ahead and removed the fd option.